resources/views/activities.blade.php

```diff
@@ -1,5 +1,8 @@
 @extends("layout.public")
 
+@section("title", __("meta.activities.title"))
+@section("description", __("meta.activities.description"))
```

Comment:
Are u sure? how long subtitle can be?
Maybe separated fields for SEO would be better?

Reply:
We're talking about activity fb tags - they say usually between 2 and 4 sentences. Currently subtitle max length is 255 characters, so it matches.

It doesn't matter whether these fields are used for SEO or for an article - in both cases we should keep them short.